### DISCUSSION TITLE
liontron switch charge and discharge ports separatly? Gone?

### DISCUSSION THREAD

**Discussion by itchako** [KG-Development/SmartBMSUtility#17] [Q&A]
hi,

installed an earlier version of this iOS App and it worked great in switching the charge and discharge Ports of my Liontron 100AH LiFePo4 battery. But when I tried some days ago to switch charging off again, which worked perfect before, it does not work anymore.... Probably due to some update work I assume.

Possible to reenable this switching option?

kind regards
itchako

**Comment by NeariX67** [comment]
I did this because some users reported that switching your (dis-)charging port would work once, but not twice, thus rendering their battery useless as they can't switch it on anymore. This resulted in many Liontron batteries being sent back to Liontron to solve that issue. As i neither want to be held reliable for this issue, nor cause any unnecessary trouble, i've implemented a "protection" that this does not happen anymore. I already thought about a small switch that would override my protection feature, maybe it's time for that one.

**Reply by itchako** [reply]
hi, 

understood. Thanks for the fast answer!

Just for your information, perhaps it helps for development:
- I used the XiaoxiangBMS iOS App many times to switch the Liontron 100AH Battery on and off.
- But it allows only to switch both, charge- and discharge port, every time off/on.
- What I am really looking for is the option to switch In- and Outgoing-MOS-Fet's separately.

kr
j

**Reply by NeariX67** [reply]
You can do this with my app. But it may be disabled for liontron batteries
